index.ts

```diff
@@ -168,6 +172,16 @@ for (const [key, data] of yamlEntries('features')) {
         }
     }
 
+    // Ensure that version ranges are only used for prehistoric features.
+    if (data.status?.support) {
+        const versions: string[] = Object.values(data.status.support);
+        if (versions.some((v) => v.startsWith('≤'))) {
```

Comment:
I don't love adding uncertainty to potentially already-certain ranges in BCD, but I expect a future resolution to https://github.com/web-platform-dx/web-features/issues/1211 to fix this at a later date, so it's fine.

Reply:
I get the discomfort. Should we instead have a single possible combination of ranged browser versions as described in https://github.com/web-platform-dx/web-features/issues/1018#issuecomment-2087553410?

Reply:
Let's leave this for now too. I think bubbling up BCD's uncertainty will mess with a bunch of values anyway (especially dates), so let's sort that out first.

Reply:
Alright, let me make some cosmetic changes around the Edge 12 date and land this.